easycv/hooks/wandb.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import cv2
+import numpy as np
+from mmcv.runner.dist_utils import master_only
+from mmcv.runner.hooks import HOOKS
+from mmcv.runner.hooks import WandbLoggerHook as _WandbLoggerHook
+from PIL import Image as PILImage
+
+
+@HOOKS.register_module()
+class WandbLoggerHookV2(_WandbLoggerHook):
+
+    def visualization_log(self, runner):
+        visual_results = runner.visualization_buffer.output
```

Comment:
how about using a dict for visualization buffer,  of which key is the image name, and value is the visualization image,  this may be more simple and sufficient for visualization